packages/cli/src/wot-servient-schema.conf.json

```diff
@@ -0,0 +1,99 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "servient": {
+            "$id": "#root/servient",
+            "type": "object",
+            "required": ["clientOnly", "scriptAction"],
+            "properties": {
+                "clientOnly": {
+                    "$id": "#root/servient/clientOnly",
+                    "type": "boolean",
+                    "default": false
+                },
+                "staticAddress": {
+                    "$id": "#root/servient/staticAddress",
+                    "type": "string"
+                },
+                "scriptAction": {
+                    "$id": "#root/servient/scriptAction",
+                    "type": "boolean",
+                    "default": false
+                }
+            }
+        },
+        "http": {
+            "$id": "#root/http",
+            "type": "object",
+            "properties": {
+                "port": {
+                    "$id": "#root/http/port",
+                    "type": "integer"
+                },
+                "proxy": {
+                    "$id": "#root/http/proxy",
+                    "type": "object",
+                    "properties": {}
+                },
+                "allowSelfSigned": {
```

Comment:
serverKey and serverCert are missing

Reply:
and they have a cross dependency

Reply:
I sadly do not have one but [here](https://github.com/eclipse/thingweb.node-wot/blob/master/packages/binding-http/src/http-server.ts#L98) you can see that they are both needed if one is needed